src/components/SignUp/SignUpFooter.tsx

```diff
@@ -92,8 +92,8 @@ const SignUpFooter: React.FC<{
                 ? privacyAgree
                 : () => setLandingStep(2)
             }
+            style={{ background: colors.darkGreen, border: colors.darkGreen }}
```

Comment:
We want to let the button style come from the shared component @HereComePretzel is creating - no need to worry about it here. In general, avoid adding `style` attributes on things manually